x/staking/client/cli/tx.go

```diff
@@ -1,26 +1,406 @@
 package cli
 
 import (
+	"bytes"
+	"encoding/hex"
+	"errors"
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/maticnetwork/bor/common"
+
+	// "github.com/maticnetwork/heimdall/bridge/setu/util"
+	"github.com/maticnetwork/heimdall/contracts/stakinginfo"
+	"github.com/maticnetwork/heimdall/helper"
+	hmTypes "github.com/maticnetwork/heimdall/types/common"
 	"github.com/maticnetwork/heimdall/x/staking/types"
 )
 
+var logger = helper.Logger.With("module", "staking/client/cli")
+
 // GetTxCmd returns the transaction commands for this module
 func GetTxCmd() *cobra.Command {
-	cmd := &cobra.Command{
+	stakingTxCmd := &cobra.Command{
 		Use:                        types.ModuleName,
 		Short:                      fmt.Sprintf("%s transactions subcommands", types.ModuleName),
 		DisableFlagParsing:         true,
 		SuggestionsMinimumDistance: 2,
 		RunE:                       client.ValidateCmd,
 	}
 
-	// this line is used by starport scaffolding # 1
+	stakingTxCmd.AddCommand(
+		SendValidatorJoinTxCmd(),
```

Comment:
I am not comfortable with `Send` prefix for commands here. Any reason to use `Send`? We can just `ValidatorJoinTxCmd()` or `NewValidatorJoinTxCmd()` or even better as `CmdValidatorJoinTx`

Reply:
Yes, I feel `ValidatorJoinTxCmd` is best. I have made changes accordingly.